frontend/src/main/pages/AdminJobsPage.js

```diff
@@ -4,7 +4,9 @@ import JobsTable from "main/components/Jobs/JobsTable";
 import { useBackend } from "main/utils/useBackend";
 import Accordion from 'react-bootstrap/Accordion';
 import TestJobForm from "main/components/Jobs/TestJobForm";
-import JobComingSoon from "main/components/Jobs/JobComingSoon";
+//import JobComingSoon from "main/components/Jobs/JobComingSoon";
```

Comment:
Should remove commented out code (in general)... unless there is a very specific reason to keep it.